### DISCUSSION TITLE
Grouping issues by tag

### DISCUSSION THREAD

**Discussion by stefano-elysium** [getsentry/support#88] [Support] [answered]
Dear team,
I managed to add a bunch of tags to my errors, however they are grouped by stack trace. This is a problem because I'm doing Sentry.captureException manually and its grouping ALL the errors together. How can I make it auto-group by tag (ex. responseid) ?

**Comment by JLuse** [comment, marked as answer]
Hey @stefano-elysium - something that might work best for grouping by tags would be to use [fingerprint rules](https://docs.sentry.io/product/sentry-basics/grouping-and-fingerprints/), specifically grouping by the [tags.tag_name](https://docs.sentry.io/product/data-management-settings/event-grouping/fingerprint-rules/#tagstag_name).
